docs/api-docs/tax/tax-rates-zones.mdx

```diff
@@ -454,6 +504,7 @@ To delete tax rates, send a request to the [Delete tax rates](/docs/rest-managem
 - [Update tax zones](/docs/rest-management/tax-rates-and-zones/tax-zones#update-tax-zones)
 - [Get tax zones](/docs/rest-management/tax-rates-and-zones/tax-zones#get-tax-zones)
 - [Delete tax zones](/docs/rest-management/tax-rates-and-zones/tax-zones#delete-tax-zones)
+- [Check tax zone](/docs/rest-management/tax-rates-and-zones/tax-zone-check#check-tax-zone)
 
 ### Tax Rates reference
```

Comment:
```suggestion
### Tax rates reference
```

Reply:
That's the name of the API (and it's in the UI), so I might keep it capitalized

Reply:
okay